src/components/GalaxyModal.js

```diff
@@ -327,9 +327,11 @@ export const GalaxyModalBase = withDisplayName('GalaxyModal')(
               value: disabled ? currentDataDisk.diskType : dataDisk.diskType,
               isDisabled: disabled,
               onChange: ({ value }) => updateDataDisk('diskType', value),
+              menuPlacement: 'auto',
```

Comment:
Nice improvement! I'm very particular about my browser window sizes, but I was able to test this by opening the inspector, which allows me to change the effective height of the browser window without actually resizing the window.